airbyte-integrations/connectors/source-woocommerce/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.2.7"
+name = "source-woocommerce"
+description = "Source implementation for Woocommerce."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/woocommerce"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "source_woocommerce"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "0.88"
```

Comment:
Why do we hardcode from the minor version?

Reply:
@maxi297 told me it was better to keep the connector stable in case of future changes.

Reply:
This was decided [here](https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1713548724372399)

It seems like we are now reconsidering this [here](https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1715906676083989)